persist/sqlite/init.sql

```diff
@@ -278,6 +291,19 @@ CREATE TABLE v2_block_transactions (
 CREATE INDEX v2_block_transactions_block_id_index ON v2_block_transactions(block_id);
 CREATE INDEX v2_block_transactions_transaction_id_block_id ON v2_block_transactions(transaction_id, block_id);
 
+CREATE TRIGGER delete_orphan_v2_transaction
```

Comment:
Little of PTSD when it comes to using triggers 😅 . Could you elaborate why the database schema can't be a chain of entities that depend upon each other, and eventually a block that can be referenced and have a `CASCADE DELETE` when we revert the chain index at a certain height? We can probably have `v2_transactions` reference `v2_block_transactions` instead of the other way around and add a `CASCADE DELETE` there?